docs/pages/dev-guide/developer-guide.md

```diff
@@ -1254,21 +1256,21 @@ pods:
 ```
 
 
-* `EXTERNAL_ROUTES` and `INTERNAL_ROUTES`
+1. Delete these 3 labels from your marathon json:
+    * `DCOS_FRAMEWORK_NAME`
+    * `DCOS_SERVICE_PORT_INDEX`
+    * `DCOS_SERVICE_SCHEME`
+
+1. `EXTERNAL_ROUTES` and `INTERNAL_ROUTES`
 
     * These have a 1:1 mapping (they are both comma separated lists). There is one internal route for every external route.
```

Comment:
The `EXTERNAL_ROUTES` and `INTERNAL_ROUTES` have a 1:1 mapping (they are both comma-separated lists).